### PR TITLE
Release version 6.1.0

### PR DESCRIPTION
**Pull request by jaylinski**
:new: :package:

### CODE REVIEW

**Review by drupol — APPROVED** [review]
All good!

File: .github/workflows/continuous-integration.yml [hunk 1]
```diff
@@ -47,20 +47,16 @@ jobs:
...
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: composer install --no-progress --no-suggest --prefer-dist --optimize-autoloader
 
       - name: Run Grumphp
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: vendor/bin/grumphp run --no-ansi -n
 
       - name: Send PSALM data
-        if: ${{ matrix.php-versions != '8.0' }} # Remove this as soon as `drupol/php-conventions` supports PHP 8
         run: vendor/bin/psalm --shepherd --stats
```
Comment (jaylinski) [inline .github/workflows/continuous-integration.yml:56]: @drupol There is an error in the `Send PSALM data` step, I don't know if this should work...
```
  vendor/bin/psalm --shepherd --stats
  shell: /usr/bin/bash -e {0}
/home/runner/work/_temp/2a3a4068-c3b0-4825-ab78-2293ac5a2797.sh: line 1: vendor/bin/psalm: No such file or directory
Error: Process completed with exit code 127.
```

Comment (drupol) [reply .github/workflows/continuous-integration.yml:56]: yes, please update with: https://github.com/loophp/collection/blob/master/.github/workflows/static-analysis.yml#L60

And don't forget to add `vimeo/psalm` in `composer.json`

**Comment by drupol** [comment]
One last run of Grumphp and we're good.

`./vendor/bin/grumphp --testsuite=cs`

**Comment by drupol** [comment]
Beautiful !

**Comment by jaylinski** [comment]
@shulard Ready for merging and tagging :)

**Comment by shulard** [comment]
Hello !
I've just tagged the v6.1.0 release. I'll try to write a better release note soon ^^

**Pull request merged**